src/GitVersionCore/GitVersionException.cs

```diff
@@ -0,0 +1,32 @@
+﻿#region License
+
+// --------------------------------------------------
```

Comment:
@asbjornu where is this license coming from?

Reply:
Good question. It's from ReSharper, but I thought GitVersion's DotSettings should override it? Either way, it's gone.